bokeh/plotting_helpers.py

```diff
@@ -469,7 +469,7 @@ def _new_xy_plot(x_range=None, y_range=None, plot_width=None, plot_height=None,
 
 
 def _handle_1d_data_args(args, datasource=None, create_autoindex=True,
-        suggested_names=[]):
+        suggested_names=None):
```

Comment:
OK, I think it used to be. Looking closer I think we could actually remove all that `suggested_names` junk entirely, tho that could go in a different PR if you prefer.